code/modules/clothing/clothing_vr.dm

```diff
@@ -124,7 +124,8 @@
 	sprite_sheets = list(
 		SPECIES_TESHARI = 'icons/mob/species/seromi/suit.dmi',
 		SPECIES_VOX = 'icons/mob/species/vox/suit.dmi',
-		SPECIES_WEREBEAST = 'icons/mob/species/werebeast/suit.dmi')
+		SPECIES_WEREBEAST = 'icons/mob/species/werebeast/suit.dmi',
+		SPECIES_MANTID_ALATE = 'icons/mob/species/mantid/onmob_suit_alate.dmi')
```

Comment:
This should probably be renamed 'suit.dmi' for consistency sake

Reply:
fixed by removing onmob_ from everything